.github/CHANGELOG.md

```diff
@@ -14,7 +14,7 @@
 
 <h3>Internal changes ⚙️</h3>
 
-- Skip Identity operation in Lightning Qubit and removed assert for applying Identity gate with more than 1 wire.
+- Skip Identity operation in Lightning Qubit and removed assert for applying Identity gate not equal to 1 wire.
```

Comment:
minor: we could add this to "Improvements" as it extends the Identity gate op in LQ (C++). This will be required for https://github.com/PennyLaneAI/catalyst/pull/1907